src/apis/impl/AuthenticationApiImpl.ts

```diff
@@ -135,19 +152,19 @@ export class AuthenticationApiImpl implements AuthenticationApi {
           delegate
         );
         //TODO Return which delegates were warned to share back info & add retry mechanism
-        if (!accessNotification) throw new Error(`Cannot create a notification to Healthcare Professional with id ${delegate}`)
+        if (!accessNotification) throw this.errorHandler.createErrorWithMessage(`Cannot create a notification to Healthcare Professional with id ${delegate}`)
```

Comment:
Here, I would do a log instead : Otherwise, it means that the procedure for the other hcps will be stopped as well. 
I would therefore do something like `console.log('...')`
And for the message `iCure could not create a notification to healthcare party ${hcp.id} to ask access back to ${loggedUser.patientId} data. Make sure to create a notification for the healthcare party so that he gives back access to ${loggedUser.patientId} data. `